test/extension/main.go

```diff
@@ -50,6 +52,8 @@ var (
 	logOptions      = logs.NewOptions()
 )
 
+// +kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;list;watch;patch;update
```

Comment:
I think we should only use this if we have a corresponding makefile setup. 
I'm fine either way, currently we don't have make targets for the test extension (this would be the first case where we need one).

If you want to add one to generate rbac, let's move the rbac files in a separate folder like we do for our controllers and let's add the target into the top-level Makefile instead of creating a new one.